src/ai/backend/test/templates/template.py

```diff
@@ -36,15 +42,22 @@ async def test(self) -> None:
 
 class TestTemplate(ABC):
     @final
-    def with_wrappers(self, *wrappers: WrapperTestTemplateProtocol) -> "TestTemplate":
+    def with_wrappers(
+        self, *wrappers: WrapperTestTemplateProtocol | type[WrapperTestTemplateProtocol]
```

Comment:
Please avoid using type unions where a specific interface should be used in a fixed manner as much as possible.